### ISSUE TITLE
Add support for GPU on Windows to Docker Desktop Kubernetes

### ISSUE THREAD

**Issue by ShuaiShao93** [docker/roadmap#654]
**Tell us about your request**
It looks like Docker Desktop supports GPU on Windows with WSL2, but the Kubernetes in it doesn't

**Which service(s) is this request for?**
Kubernetes in Docker Desktop

**Tell us about the problem you're trying to solve. What are you trying to do, and why is it hard?**
We want to run a local k8s cluster on Windows workstation, but there is no way to use its GPU

**Are you currently working around the issue?**
No

**Comment by peter-wangxu** [comment]
Looking forward to..
which benefit the AI workloads running on docker-desktop with kubernetes

**Comment by luancaarvalho** [comment]
Hello guys !

Do you know if the Kubernetes deployed by the docker desktop continue without support for GPU?

**Comment by beneiltis** [comment]
Yeah i also wonder why this couldnt be a usecase without WSL2. It would be great to utilze my GPU as easy as i can make use of my Docker Desktop Kubernetes.

**Comment by ospf2fullstack** [comment]
Yes.. desktop PC's don't support TCC mode, which is what kubernetes is looking for, but in my case, WDDM is enforced and cannot be overwritten while a display is attached... As soon as I run  `kubectl label node docker-desktop nvidia.com/gpu.present=true --overwrite`, it reverts back to false and I believe it's due to the `mode` being enforced. 
```
   Driver Model
        Current                           : WDDM
        Pending                           : WDDM
```

**Comment by trylovetom** [comment]
Is there any update? Otherwise, I’ll have to switch to using Minikube.

**Comment by ospf2fullstack** [comment]
> Is there any update? Otherwise, I’ll have to switch to using Minikube.

@trylovetom , you can use docker compose and reference the GPU allocation for your pod, just not with the kubernetes integration. not sure how "critical" it is, but this could be a workaround - unless you're hoping to practice the deployment(s) with helm/yaml directly.
